src/components/HeartbeatBar.vue

```diff
@@ -111,6 +98,47 @@ export default {
             return placeholders.concat(this.beatList.slice(start));
         },
 
+        values() {
+            if (!this.shortBeatList || !this.shortBeatList.length) {
+                return [];
+            }
+
+            const valueObj = {};
+
+            this.shortBeatList.forEach(({ status, time }) => {
+                const date = dayjs(time).format("YYYY-MM-DD");
+
+                if (valueObj[date] === undefined) {
+                    valueObj[date] = 0;
+                }
+
+                let count = 0;
+
+                switch (status) {
+                    case 0:
+                        break;
+                    case 1:
+                        count = 10;
+                        break;
+                    case 2:
+                        count = 2;
+                        break;
+                    case 3:
+                        count = 5;
+                }
+
+                valueObj[date] += count;
```

Comment:
Sorry to say, but I find this very confusing:
What is a `XYZ counts` in 
![image](https://github.com/louislam/uptime-kuma/assets/26258709/805820f0-c83e-4888-ac13-0ca38e34e1e8)

Reply:
added percentage data.